openamp/libmetal.cmake

```diff
@@ -54,6 +54,7 @@ if("${CONFIG_ARCH}" STREQUAL "sim")
   set(LIBMETAL_ARCH x86_64)
 elseif("${CONFIG_ARCH}" STREQUAL "risc-v")
   set(LIBMETAL_ARCH riscv)
+  set(HAVE_STDATOMIC_H 1)
```

Comment:
should we enable HAVE_STDATOMIC_H for all arch?

Reply:
Sorry I have little idea about toolchains I am not using now.

Reply:
@xiaoxiang781216 could you please resolve this conversation? Or point to some possible solution?

Reply:
I am waiting https://github.com/apache/nuttx/pull/12516

Reply:
let's close this patch, @yf13 please try https://github.com/apache/nuttx/pull/12516.